### ISSUE TITLE
📣 Elementor Pro v3.0 Beta 3 Release 📣

### ISSUE THREAD

**Issue by shilo-ey** [elementor/elementor#12054]
===========

# [Beta 3 UPDATED - PLEASE READ BELOW!](https://github.com/elementor/elementor/issues/12054#issuecomment-675682749)

===========

Hi Elementor Beta testers 👋,

Last month, we announced [the beta release of Elementor 3.0](https://github.com/elementor/elementor/issues/11821), titled ‘_Design System Features_’. 
This release is divided into 2 phases:

- PHASE 1 - Global Colors, Global Typography & Site Settings
- PHASE 2 - The new and revamped Theme Builder

It’s time for the second phase…

**Before you test, please make sure you are using:**

- Elementor Core v3.0 beta5
- Elementor Pro v3.0 beta2

# Introducing the Brand New Theme Builder: Manage Your Entire Site in a Simple and Intuitive Way

![3 0_Pro_release_cover](https://user-images.githubusercontent.com/45009641/89620984-393b3980-d899-11ea-9524-4a26f58f836e.jpg)

Elementor has established itself as the standard website builder solution ever since the launch of Elementor 2.0, back in 2018.

Professionals know they can edit every part of their website, visually and without code, using Elementor.

BUT… Being the standard didn’t leave us complacent.

In fact, we are reinventing the WordPress website building experience.

## What’s new in this Theme Builder?
The new Theme Builder is a React-based application, which provides an overview of the site, including all the site parts: header, footer, global page template, global post template and more. Every part of the site is intuitively within reach, making the task of designing a complete website that much easier.

Your confused days are over! The new Theme Builder will allow you to get a glance over your entire site structure visually. See which parts are active and which are not, manage which template applies to different areas of your website, and have easy access to change, update or add content as you please.

Each site part gets an automatic image preview and a screenshot of the actual template. This lets you get a visual ‘birds-eye-view’ of your entire site, so you can pinpoint the exact template you wish to edit.

Confused about what each part does? Whether its Archive, search results or single post… If you or your client are confused, we added tooltips to help you know the role and logic of each part.

## How It Works
This next paragraph is going to turn you into a ‘super-Elementor-builder’, ahead of most web creators, so I suggest you pay special attention.

To start building your site using Theme Builder, click on the hamburger icon on the top left of the editor, then click on ‘Theme Builder’.

You will now see all the parts of your site. 

If it’s a new site, and it doesn’t have any parts yet, click on any part and build it. We suggest starting with a new header (just to stay ‘on top’ of things…)

After you build the header, you can go back to the Theme Builder screen, and see where this header is applied across the site.

#### Visually access, edit, rename and delete any part of your site
The Theme Builder revamp dramatically improves your site building experience. It also improves how you manage and update your site. Basically, it’s your visual map of the site.

#### Decide where each site part appears
You can now access Display Conditions right from the Theme Builder screen. A great example of its usefulness is if you want 2 headers, one for your blog and the other for the rest of the site. Finally, control all your site conditions from one place! The days you had to edit a header just to change its instances are over. From now on, you can change, add or remove display conditions without leaving the Theme Builder

#### Import and export any site part 
If you want to use a site part on another site, quickly import and export it from the Theme Builder.

#### Which site part is live or has conditions?
When managing a large-scale site, it’s sometimes difficult to get a grip over all the templates on the site. I’m sure you know what I’m talking about.

Theme Builder solves this, by adding a green dot next to the site parts that have conditions. This way, you know which templates are active and which are drafts.

## Design System - The Structure Part
With global colors, typography and structure, you now have more control over your site’s design system.

Global colors allow you to set your color scheme once, and complete a full site design with the same colors, consistently.
Same goes for global typography.

For your site structure, a similar approach applies. You build your site parts in one place, and assign them wherever you want. Instead of building single templates that are spread across multiple places and are impossible to manage, every part of your site is viewed from one visual Theme Builder.

This is the smart way of building websites, and we see it as a big improvement in the workflow of our users.

#### Test Flows:

**A: Make sure all access points to Theme Builder works properly**
1. Set up an environment with Elementor Pro installed, with v2.9.x installed (Core and Pro), and create several different Site Parts (Header, Footers, Single Post templates, Single Page templates etc)
2. Upgrade to v3.0 Pro Beta 1 and make sure Elementor Core v3.0 Beta 4 is installed
3. Click `CMD/CTRL + Shift + E` to open the new Theme Builder from any screen
4. Make sure the screen opens up and responds to any of your interactions
5. Close the Theme Builder, go ahead and edit any page or post with Elementor
6. In the Editor, click on the menu (hamburger) icon and click “Theme Builder”
7. Make sure the screen opens up and responds to any of your interactions
8. Close the Theme Builder, go ahead and view the page that you are in
9. In the frontend view of your page, go to the top admin bar, hover over the “Edit with Elementor” item and click on “Open Theme Builder” item
10. Make sure the screen opens up and responds to any of your interactions
11. Create a new Site Part, and publish it, click on “Open Theme Builder” button which located in the publish message (Editor toaster)
12. Make sure the screen opens up and responds to any of your interactions

**B: Make sure all previous Site Parts are located under the correct tab**
1. Open Theme Builder by using one of the ways mentioned in test flow A
2. Make sure you can see all of your previously created Site Parts under “All Parts” tab
3. Make sure that under “All Parts” tab you can see automatically-generated screenshots (generated automatically on publish/update) or a fallback image based on Site Part type
4. Go to each tab and make sure you can find the corresponding Site Parts there

**C: Make sure Add New screen and it’s contents functioning**
1. Open Theme Builder by using one of the ways mentioned in test flow A
2. If you haven’t created any Site Part yet, the Add New screen should be visible without any interaction
3. If you created Site Parts already, the Add New screen should be available by clicking on “Add New” button, in the top right corner, when in the “All Parts” tab
4. Click on the ‘i’ icon and make sure an educational modal has opened. Make sure you can view the video, and try opening the modal for each Site Part
5. Close the Educational modal and click on any Site Part you want to create, for example Header
6. You will be redirected to the corresponding Part editing screen and would be able to use the Library to import any template
7. Publish this part with several display conditions and go back to the Theme Builder
8. Go the All Parts tab and the specific Site Part tab and make sure it’s visible there

**D: Make sure specific Site Part “Add New” button opens the corresponding part**
1. Open Theme Builder by using one of the ways mentioned in test flow A
2. Go to any Site Part tab and click on the “Add New” button and make sure it creates the correct Site Part (e.g clicked Add New under Header tab - a header should be created)
3. Publish this site part and make sure it appears in the correct place in Theme Builder
4. Go back to the Theme Builder, hover over each of the tabs and click on the circular ‘+’ icon
5. Make sure it creates the correct Site Part (e.g clicked Add New under Footer tab - a footer should be created)

**E: Make sure dynamic screenshots are being generated**
1. Using different browsers publish several Site Parts, and wait about 1 minute for the thumbnails generation to take place
2. Open Theme Builder by using one of the ways mentioned in test flow A
3. Make sure you can see the generated thumbnails for your newly created Site Parts
4. If, for any reason, your Site Parts thumbnail generation failed, you would be able to see a fallback SVG placeholder per part type
5. Please note that the auto-generated thumbnails may produce different outputs per browser
6. Add a featured image to one of your Site Parts and hit Update
7. Open Theme Builder by using one of the ways mentioned in test flow A and make sure the featured image appears in the Site Part card instead of the auto-generated thumbnail

**F: Make sure Parts preview screen works correctly and all of its properties work properly**
1. After creating several Site Parts, open the Theme Builder by using one of the ways mentioned in test flow A
2. Under “All Parts” tab hover over any of your Site Parts and click Preview
3. You’ll be directed to the corresponding Part tab and would be able to see a live view of your Site Part
4. Make sure everything works properly even when you have more than 5 parts in the same frame
5. Validate each one of the options that are available in each Site Part card:
6. Make sure the “Conditions Indicator” works properly - Site Part with any display condition should be marked with green color, Site part without any display condition applied should be marked with gray color
7. Make sure the part’s name is correct
8. Make sure the author and creation date is correct and by clicking “Edit” will move you to the editing of the part
9. Make sure the “Instances” are correct and you can edit and affect the Display conditions of each Site Part
10. Make sure that by clicking on the “More Options” button you can see the options menu appears and Rename, Delete and Export are available and works properly
11. Repeat step 5 on various Site Parts and make sure everything works properly

-----

## More Improvements

- New: Introducing the new and improved Theme Builder
- Tweak: Removed `.elementor-inner` and `.elementor-column-wrap` from DOM output to improve performance (#7351, #7817, [Developers Blog Post](https://developers.elementor.com/dom-improvements-ahead-html-wrappers-removal-from-v3-0/))
- Tweak: Added contextual anchors `ID` support to Table of Contents widgets (#10052)
- Tweak: Added WeChat and Weibo social networks to Share Buttons widget (#11554)
- Tweak: Added Dynamic capabilities for Redirect after Login/Logout in Login widget (#11343)
- Tweak: Added Blend Mode and CSS Filters controls to adjust the Background Overlay in Flipbox widget (#11653)
- Tweak: Added responsive capabilities to Toggle Button styling in Nav Menu widget (#8269)
- Tweak: Added responsive Text Alignment control in Call to Action widget (#11968)
- Tweak: Added dynamic content to Ribbon element in Call to Action widget (#10364)
- Tweak: Converted uses of Color and Typography Schemes to Global Colors and Fonts
- Tweak: Separated Title and Description control fields labels in Call to Action widget
- Tweak: Removed unnecessary style in WC Product with variations
- Tweak: Converted Portfolio, Posts and Share Buttons widgets to use CSS Variable-based Elementor Grid ([Developers Blog Post](https://developers.elementor.com/elementor-dropping-support-ie/))
- Fix: Divider alignment issue in Post Info widget (#11628)
- Fix: Color doesn’t change in Products Archive Description widget 
- Fix: WC Product variations layout breaks when using Variation Swatches plugin
- Fix: WC Product variations layout issue
- Fix: WC Product variations mobile zoom-in glitch
- Fix: Can’t edit a Popup after accessing Theme Style
- Fix: Twitter icon missing in Blockquote widget
- Fix: Removed redundant default text color from Share Buttons minimal skin
- Fix: UI glitch in Display Conditions modal
- Fix: Insert template button UI glitch in Templates Library
- Fix: Added sanitization to post titles in WordPress dashboard for better security
- Deprecated: See all deprecations to this version in our [Developers Deprecations Post](https://developers.elementor.com/v3-0-planned-deprecations/)

----

## How to Install the Beta Version
If you still haven't enabled beta testing in Elementor, please follow these steps:

- Go to Elementor > Tools > Versions
- Enable the beta tester feature
- Go to Dashboard > Updates and update to the latest beta version.

This will get you set up to test the new Elementor Pro 3.0 Beta.

---- 

## Notes
For bugs and remarks, please reply with a comment on the specific issue here on GitHub.

**Please keep this related to the Current beta Only! Any other issue or feature request, feel free to open a new issue.**

Designated time for release: `August 25, 2020` (Estimated)
 
Many thanks for your support and help!

Cheers,
_Elementor Team_

**Comment by rodolphebertozzo** [comment]
2 remarks :
- Be careful with the space between the logo and the title "Instances".
- Overviews don't necessarily load up
See below:
![at2](https://user-images.githubusercontent.com/56270626/89623376-ca5be180-d894-11ea-9218-fa2ac55251fa.PNG)
![attention](https://user-images.githubusercontent.com/56270626/89623378-caf47800-d894-11ea-9e78-4ce7073fd41f.PNG)

**Comment by pintolog** [comment]
I'm trying to get to the new Theme Builder by pressing CMD+SHIFT+E,
I see tle E logo, loading and than i get this error

![image](https://user-images.githubusercontent.com/7497319/89624310-b61ce200-d89e-11ea-88ee-014b5eca63c6.png)

**Comment by sermalefico** [comment]
Error: Method get_site_editor_type does not exist.

**Comment by itaides** [comment]
I got syntax error (ngix server php 7.4) in the first times i try to install beta . 
In the 3 time it works but there is few bugs: 
When i try to delete widget , its duplicate it ,so insted of delete its just duplicate.  
Hello theme: single post ,not working with full width.

**Comment by rodolphebertozzo** [comment]
Another problem, I don't have all my headers in the "Header" section, some of them appear in the "Single Page" section.

**Comment by pintolog** [comment]
> I'm trying to get to the new Theme Builder by pressing CMD+SHIFT+E,
> I see the E logo, loading and than i get this error
> 
> 
> https://www.loom.com/share/bfc78bf20b4c4f4f91cc30275d1be3ed
> 
> ![image](https://user-images.githubusercontent.com/7497319/89624310-b61ce200-d89e-11ea-88ee-014b5eca63c6.png)

After a quick check, the plugin that cause this bug is:
**Elementor Custom Skin Pro**

**Comment by fabianenzensperger** [comment]
Oh yes, really cool new way of creating a theme. And ALL in Elementor. The only thing missing is the pop-ups hehe, right?

Is this the site editor (now the new "Theme Builder") that was mentioned in the first beta of Elementor 3.0? Or can one expect anything in the direction of uniform global settings in relation to the layout, such as uniform spacing, sizes of icons, etc.?

Otherwise, thanks for the constant further development. It would be nice if (very) soon with some existing widgets like the blog Posts #4440 #3140 #1284 (same things please for the portfolio widget), Custom Post Types for Blog Post, Wordpress Navi #3275 , Form Fields Conditional Display Logic #2638 , more innovative sticky options, etc. to be improved.

These are all things that the community almost wants even more, as cool as the other things you do are. All of these things are in the TOP10 that are most commented on and desired. I think that's a sign, isn't it?

And most of the things are not blatant things, they are done relatively quickly. But maybe I'm wrong too.

The following are the first things I noticed about the Elementor Pro Beta 1 ...

**Comment by fabianenzensperger** [comment]
The first thing I noticed yesterday at Beta 4 for Free 3.0, and that hasn't changed with Beta 1 of Pro, is the "Open Theme Builder" option under the "Edit with Elementor" panel. Doesn't that make any sense at this point, does it?

Although you can also reach it faster if you are currently on a page and do not want to jungle your way through the Worpdress dashboard. So it's not a bug or a criticism, I just want to understand it and leave my thoughts behind. Maybe it will come in handy when you test it!

<img width="239" alt="Bildschirmfoto 2020-08-07 um 10 17 07" src="https://user-images.githubusercontent.com/57997262/89627715-91733b00-d89b-11ea-8597-10b52fc62f3d.png">

**Comment by black-eye** [comment]
My Templates > Theme Builder link in Admin Menu reffers to non extisting page (/wp-admin/elementor-app)
![obrazek](https://user-images.githubusercontent.com/5634870/89629763-aa312000-d89e-11ea-885d-7e4ab8816831.png)

**Comment by black-eye** [comment]
When I tru tu edit the Template, I got this JS error:
```

Uncaught TypeError: setting getter-only property "children"
    getContainer https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    Backbone 2
editor.min.js:2:119783
    getContainer https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    Backbone 16
    render self-hosted:935
    onServerRequestEnd https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    parseDynamicSettings https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.0.0-beta4:2
    success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    forEach self-hosted:216
    success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    Backbone 3
    success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
    forEach self-hosted:216
    success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
    success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
    Backbone 6
    send https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
    sendBatch https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
    sendBatch self-hosted:935
    Backbone 2
    addRequest https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
    loadCacheRequests https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    t https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.0.0-beta4:2
    Backbone 2
    refreshCacheFromServer https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    t https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.0.0-beta4:2
    parseDynamicSettings https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.0.0-beta4:2
    each Backbone
    parseDynamicSettings https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.0.0-beta4:2
    serializeData https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    Backbone 2
    render https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
    render self-hosted:935
    Backbone 11
    render self-hosted:935
    Backbone 7
```
Probably because of this error the inline text editing doesn't work.

**Comment by d19dotca** [comment]
For the Parts dynamic preview images... it seems like the expectation is this shouldn't take any more than a minute, however with only the basic parts defined in my test site (4 parts total, header, footer, single page, single post) it seemed like it took 10+ minutes to generate. In fact I originally thought maybe it was only updating on editing them, so I edited two of the Parts and it seemed the image was generated after that on the ones I edited, but it still took about 10-20 minutes for the dynamic images for the other parts I left alone. This seems like unexpected behaviour and I believe should be fixed. Had plenty of memory available on server, so I don't think it's any resource constraints or anything like that.

**Comment by black-eye** [comment]
My first impresions. Lets put aside that after all the hype raised and the time spent waiting I can’t help myself: it’s a bit underwhelming. It’s basicaly just a new look for the old stuff. Yes, I know UI is more than a graphic style, so let me talk the functionality. Because I feel, there are areas, where it got worse.

1. No direct access to the Theme Buider - admin menu links to 404. Hopefully it’s just a bug.
2. No possibility to export all the templates at once or selected in bulk. If there is such possibility, show me it pls.
3. No full template specifications / actions on Theme Builder (TB) “dashboard” (all templates filter). I can’t see the instances, conditions etc. When I want to edit the template (what else I’d be there for, right?), I need one extra click to get to the template “detail” and then click the edit link. I usually have only one template per each part, so I’d like to manage them right from the dashboard, not to chase them in several sections one by one.
3. Templates categories are gone.
4. Why the extra options like export live under “...” button and are not visible right away - there’s plenty of space there.
5. No info about the author, date or shortcode.
6. Is there a possibility to get to the template dashboard? To set the stuff like language, category etc.
7. I haven’t tried it on multilang site, but I have the feeling it would be a complete disaster. I have no idea, how to select the language there.

So these are my observations. To be honest, if the current “admin dashboard” style stayes in the old place, I will probably use it rather this new one. Just for the lack of the functionality of the later.

Basically the most (and for me only) exiting addition are the template thumbnails. Unfortunately they are not applied to the non-theme templates (yet??), where they would be much more needed.

**Comment by d19dotca** [comment]
> No direct access to the Theme Buider - admin menu links to 404. Hopefully it’s just a bug.

I don't seem to have that issue in mine, interestingly enough. I wonder what that could be, maybe another plugin or perhaps your core isn't updated to beta 4? Just guessing.

**Comment by rodolphebertozzo** [comment]
> My first impresions. Lets put aside that after all the hype raised and the time spent waiting I can’t help myself: it’s a bit underwhelming. It’s basicaly just a new look for the old stuff. Yes, I know UI is more than a graphic style, so let me talk the functionality. Because I feel, there are areas, where it got worse.
> 
> 1. No direct access to the Theme Buider - admin menu links to 404. Hopefully it’s just a bug.
> 2. No possibility to export all the templates at once or selected in bulk. If there is such possibility, show me it pls.
> 3. No full template specifications / actions on Theme Builder “dashboard” (all templates filter). I can’t see the instances, conditions etc. When I want to edit the template (what else I’d be there for, right?), I need one extra click to get to the template “detail” and then click the edit link. I usually have only one template per each part, so I’d like to manage them right from the dashboard, not to chase them in several sections one by one.
> 4. Templates categories are gone.
> 5. Why the extra options like export live under “...” button and are not visible right away - there’s plenty of space there.
> 6. No info about the author, date or shortcode.
> 7. Is there a possibility to get to the template dashboard? To set the stuff like language, category etc.
> 8. I haven’t tried it on multilang site, but I have the feeling it would be a complete disaster. I have no idea, how to select the language there.
> 
> So these are my observations. To be honest, if the current “admin dashboard style” stays in the old place, I will probably use it rather than this new one. Just for the lack of the functionality of the later.
> 
> Basically the most (and for me only) exiting addition is the template thumbnails. Unfortunately they are not applied to the non-theme templates (yet??), where they would be much more needed.

@black-ey, let's not forget that for now it's Beta and not RC, suggesting that the new builder theme is still under development and that the missing features you mentioned will probably be implemented in the next few days

**Comment by avocadesign** [comment]
I've got a bug where custom uploaded fonts are not recognised. They are there in the Elementor custom font backend but not showing as an option to use in Elementor since beta4.

**Comment by fabianenzensperger** [comment]
One more note: I think a search field to find various sections, for example on very large websites, is absolutely necessary. There is also a search in the WP dashboard.

I think that the Theme Builder is only ready for use when all the functions that the WP Dashboard provides are in place.

**Comment by fabianenzensperger** [comment]
> My first impresions. Lets put aside that after all the hype raised and the time spent waiting I can’t help myself: it’s a bit underwhelming. It’s basicaly just a new look for the old stuff. Yes, I know UI is more than a graphic style, so let me talk the functionality. Because I feel, there are areas, where it got worse.
> 
> 1. No direct access to the Theme Buider - admin menu links to 404. Hopefully it’s just a bug.
> 2. No possibility to export all the templates at once or selected in bulk. If there is such possibility, show me it pls.
> 3. No full template specifications / actions on Theme Builder “dashboard” (all templates filter). I can’t see the instances, conditions etc. When I want to edit the template (what else I’d be there for, right?), I need one extra click to get to the template “detail” and then click the edit link. I usually have only one template per each part, so I’d like to manage them right from the dashboard, not to chase them in several sections one by one.
> 4. Templates categories are gone.
> 5. Why the extra options like export live under “...” button and are not visible right away - there’s plenty of space there.
> 6. No info about the author, date or shortcode.
> 7. Is there a possibility to get to the template dashboard? To set the stuff like language, category etc.
> 8. I haven’t tried it on multilang site, but I have the feeling it would be a complete disaster. I have no idea, how to select the language there.
> 
> So these are my observations. To be honest, if the current “admin dashboard style” stays in the old place, I will probably use it rather than this new one. Just for the lack of the functionality of the later.
> 
> Basically the most (and for me only) exiting addition is the template thumbnails. Unfortunately they are not applied to the non-theme templates (yet??), where they would be much more needed.

One more note: I think a search field to find various sections, for example on very large websites, is absolutely necessary. There is also a search in the WP dashboard.

I think that the Theme Builder is only ready for use when all the functions that the WP Dashboard provides are in place.

**Comment by shilo-ey** [comment]
> 2 remarks :
> 
> * There are some display problems as you can see below. I tested with 3 different browsers and this UI problem persists, even with other users.
> * Overviews don't necessarily load up
@rodolphebertozzo Thanks

Auto Screenshots mechanism depends on the client's browser, which can result in failure in creating screenshots.

The next beta should include a live preview (iframe) in the inner page of a part, so for example when you will click on a header, it will redirect you to the Headers tab and will scroll to the corresponding part.




[**@pintolog**](https://github.com/pintolog) commented on [Aug 7, 2020, 11:11 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670392054 "2020-08-07T08:11:52Z - Replied by Github Reply Comments"):
> I'm trying to get to the new Theme Builder by pressing CMD+SHIFT+E,
> I see the E logo, loading and than i get this error
@sermalefico 
Thanks, we are contacting the creators of the large Elementor addons for better compatibility



[**@itaides**](https://github.com/itaides) commented on [Aug 7, 2020, 11:20 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670395372 "2020-08-07T08:20:09Z - Replied by Github Reply Comments"):
> I got syntax error (ngix server php 7.4) in the first times i try to install beta . 
> In the 3 time it works but there is few bugs: 
> When i try to delete widget , its duplicate it ,so insted of delete its just duplicate.  
> Hello theme: single post ,not working with full width.

Can you please share your system info?


[**@Rodolphe Bertozzo**](https://github.com/rodolphebertozzo) commented on [Aug 7, 2020, 11:24 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670397284 "2020-08-07T08:24:41Z - Replied by Github Reply Comments"):
> Another problem, I don't have all my headers in the "Header" section, some of them appear in the "Single Page" section.

Do you recall what is the action that you've done to make them move location?

Steps that we can reproduce this?


[**@fabianenzensperger**](https://github.com/fabianenzensperger) commented on [Aug 7, 2020, 11:44 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670405721 "2020-08-07T08:44:20Z - Replied by Github Reply Comments"):
> Oh yes, really cool new way of creating a theme. And ALL in Elementor. The only thing missing is the pop-ups hehe, right?

The Theme Builder is the way for us to guide even non-experienced users on what are the essential parts that are required for creating a Complete website structure. 

Popups are a different thing, that would be handled in a similar manner soon.

> Is this the site editor (now the new "Theme Builder") that was mentioned in the first beta of Elementor 3.0? Or can one expect anything in the direction of uniform global settings in relation to the layout, such as uniform spacing, sizes of icons, etc.?

We are heading towards improving the workflows in the last releases, and will continue doing so. we will also add highly requested features.

> Otherwise, thanks for the constant further development. It would be nice if (very) soon with some existing widgets like the blog Posts #4440 #3140 #1284 (same things please for the portfolio widget), Custom Post Types for Blog Post, Wordpress Navi #3275 , Form Fields Conditional Display Logic #2638 , more innovative sticky options, etc. to be improved.
> 
> These are all things that the community almost wants even more, as cool as the other things you do are. All of these things are in the TOP10 that are most commented on and desired. I think that's a sign, isn't it?
> 
> And most of the things are not blatant things, they are done relatively quickly. But maybe I'm wrong too.
> 
> The following are the first things I noticed about the Elementor Pro Beta 1 ...

Most of these things are in our roadmap and **will** be dealt off soon enough.




[**@black-eye**](https://github.com/black-eye) commented on [Aug 7, 2020, 12:12 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670418342 "2020-08-07T09:12:32Z - Replied by Github Reply Comments"):
> My Templates > Theme Builder link in Admin Menu reffers to non extisting page (/wp-admin/elementor-app)
> 
> ![obrazek](https://user-images.githubusercontent.com/5634870/89629763-aa312000-d89e-11ea-885d-7e4ab8816831.png)
> 
> EDIT: This link should be preserved in the 3.0 (removing it from the menu wouldn't be a good solution). We definitely need a straight way to get to the Theme Builder directly, without the need to enter some random Elementor page first. 
> 
> Imagine you use Elementor just for the headers, footers, and blog layout, and all the posts are edited in classic editor, so you don't have any page in Elementor (yes, I know, very unlikely, but, it is just an example). With this setup you are unable to enter the Theme Builder, until you create some "fake" Elementor page just for this purpose.

The page won't be removed in the near future. we will promote there the New Theme Builder and will direct to it from the new links (Editor menu, admin bar menu, Publish toaster - will be released in beta3)


[**@black-eye**](https://github.com/black-eye) commented on [Aug 7, 2020, 3:59 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670503800 "2020-08-07T12:59:27Z - Replied by Github Reply Comments"):
> When I tru tu edit the Template, I got this JS error:
> ```
> 
> Uncaught TypeError: setting getter-only property "children"
>     getContainer https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     Backbone 2
> editor.min.js:2:119783
>     getContainer https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     Backbone 16
>     render self-hosted:935
>     onServerRequestEnd https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     parseDynamicSettings https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.0.0-beta4:2
>     success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     forEach self-hosted:216
>     success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     Backbone 3
>     success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
>     forEach self-hosted:216
>     success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
>     success https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
>     Backbone 6
>     send https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
>     sendBatch https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
>     sendBatch self-hosted:935
>     Backbone 2
>     addRequest https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.0.0-beta4:2
>     loadCacheRequests https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     t https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.0.0-beta4:2
>     Backbone 2
>     refreshCacheFromServer https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     t https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.0.0-beta4:2
>     parseDynamicSettings https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.0.0-beta4:2
>     each Backbone
>     parseDynamicSettings https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.0.0-beta4:2
>     serializeData https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     Backbone 2
>     render https://www.blackeye.cz/report/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.0-beta4:2
>     render self-hosted:935
>     Backbone 11
>     render self-hosted:935
>     Backbone 7
> ```
> Probably because of this error the inline text editing doesn't work.

can you please explain what are the actions that lead to this error?


[**@d19dotca**](https://github.com/d19dotca) commented on [Aug 8, 2020, 9:29 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670832374 "2020-08-08T06:29:32Z - Replied by Github Reply Comments"):
> For the Parts dynamic preview images... it seems like the expectation is this shouldn't take any more than a minute, however with only the basic parts defined in my test site (4 parts total, header, footer, single page, single post) it seemed like it took 10+ minutes to generate. In fact I originally thought maybe it was only updating on editing them, so I edited two of the Parts and it seemed the image was generated after that on the ones I edited, but it still took about 10-20 minutes for the dynamic images for the other parts I left alone. This seems like unexpected behaviour and I believe should be fixed. Had plenty of memory available on server, so I don't think it's any resource constraints or anything like that.

The new Theme Builder will create only one template screenshot (and not multiple) to avoid slow clients and overloading their memory. this is the reason that it will take a few minutes for completion.
We are working on optimizing this task.



[**@black-eye**](https://github.com/black-eye) commented on [Aug 8, 2020, 5:57 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670938473 "2020-08-08T14:57:42Z - Replied by Github Reply Comments"):
> My first impresions. Lets put aside that after all the hype raised and the time spent waiting I can’t help myself: it’s a bit underwhelming. It’s basicaly just a new look for the old stuff. Yes, I know UI is more than a graphic style, so let me talk the functionality. Because I feel, there are areas, where it got worse.
> 
> 1. No direct access to the Theme Buider - admin menu links to 404. Hopefully it’s just a bug.

Indeed, a bug.

> 2. No possibility to export all the templates at once or selected in bulk. If there is such possibility, show me it pls.

You will have to be patient for a couple of more months :)

> 3. No full template specifications / actions on Theme Builder “dashboard” (all templates filter). I can’t see the instances, conditions etc. When I want to edit the template (what else I’d be there for, right?), I need one extra click to get to the template “detail” and then click the edit link. I usually have only one template per each part, so I’d like to manage them right from the dashboard, not to chase them in several sections one by one.

This screen is designed for a general overview. the specific part screen should show you a live preview and it's actions.

> 3. Templates categories are gone.

If you wish to use categories you can still use the backend screen. although, as you mentioned, when having one part from each type - why would you need it?

> 4. Why the extra options like export live under “...” button and are not visible right away - there’s plenty of space there.

It's not an action that you will make on a daily basis and we rather add one more click to make sure you wish to do it.

> 5. No info about the author, date or shortcode.

You should see the author name, and the date. can't you see them in the inner screen?

> 6. Is there a possibility to get to the template dashboard? To set the stuff like language, category etc.

Please elaborate. what is the template dashboard? WP dashboard post?
If so, as mentioned, the "Legacy" Theme builder will still be there.

> 7. I haven’t tried it on multilang site, but I have the feeling it would be a complete disaster. I have no idea, how to select the language there.

Same as 6.

> So these are my observations. To be honest, if the current “admin dashboard style” stays in the old place, I will probably use it rather than this new one. Just for the lack of the functionality of the later.
> 
> Basically the most (and for me only) exiting addition is the template thumbnails. Unfortunately they are not applied to the non-theme templates (yet??), where they would be much more needed.

Yet.




[**@avocadesign**](https://github.com/avocadesign) commented on [Aug 10, 2020, 7:57 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-671161501 "2020-08-10T04:57:03Z - Replied by Github Reply Comments"):
> I've got a bug where custom uploaded fonts are not recognised. They are there in the Elementor custom font backend but not showing as an option to use in Elementor since beta4.

Known issue that will be dealt



Thanks for your feedback! 

Pro Beta 2 is soon to be released alongside with Core Beta 5, stay tuned.

**Comment by rodolphebertozzo** [comment]
Change Milestone for this : https://github.com/elementor/elementor/issues/4417

**Comment by rodolphebertozzo** [comment]
And for shortcodes ?

**Comment by shilo-ey** [comment]
Hey all,

Please check this version thoroughly and report it here.

**This beta5 version includes the following changes:**

- Tweak: Improved screenshots mechanism
- Tweak: Added live view of Site Parts in specific Site Parts screen
- Fix: UI glitches in Theme Builder
- Fix: Changed "Product Archive" to "Products Archive" in Theme Builder tabs


**Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/pojome/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**

Thanks again to everyone.

**Comment by rodolphebertozzo** [comment]
Add this in Milestone: https://github.com/elementor/elementor/issues/11343

**Comment by rodolphebertozzo** [comment]
@shilo-ey To answer you, here are 2 screenshots (header templates), one with the old theme builder and the next one with the new one. We can see that there are 5 headers recognized by Elementor with the old theme constructor and that since the new interface, the 4 other headers are considered as "simple pages".
![pro1](https://user-images.githubusercontent.com/56270626/89815347-e9809a80-db44-11ea-974c-804e6132176d.PNG)
![pro2](https://user-images.githubusercontent.com/56270626/89815354-eab1c780-db44-11ea-8e72-08ab606fcaee.PNG)

**Comment by rodolphebertozzo** [comment]
2 other remarks :

-Add a line preview option for each model
-Add a window when you create a template to be able to directly give a name to the template

**Comment by jonathansoltero** [comment]
How often do the global parts screenshots update? I am still seeing screenshots from yesterday.

I would remove the Templates tab & put Theme Builder under Elementor tab so it remains clear that it's part of Elementor.

**Comment by fabianenzensperger** [comment]
@shilo-ey How can I actually change a template created with the Theme Builder? For example converting a "Single Page" template into a "Header" template?

And what would be MUCH better if header and navigation were separate. So you could leave the navigation to the navigation and place a separate header (for example image with text) on the pages.

That would make it a lot easier, wouldn't it? Maybe you could think about it for the new Theme Builder. I am happy about your thoughts on it ...

**Comment by gravnetic** [comment]
I have custom classes on objects and I am unable to access the editor now.
Undefined variable: css
I am able to access the editor in safe mode.

**Comment by hybridmuse** [comment]
Design system's colors and fonts only show when the Site Settings view is active.
I save changes but as soon as I quit that mode all styles disappear. Going back to Site Settings they reappear.
Using latest betas.

**Comment by mrdumont** [comment]
For me, the Error: Method get_site_editor_type does not exist error turned out to be a conflict with the 'Ele Custom Skin' plugin (Elementor Custom Skin for Posts and Archive Posts). Disabling the plugin solved the problem.

First impression - looks promising.

**Comment by poziomapl** [comment]
In version 3.0 Pro in Theme Builder it is not possible to add a translation for a theme page, e.g. with PolyLang or WPML for headers/footers/archives/single pages. Previously, the default Wordpress panel allowed this, and the new Theme Builder layout does not - i this is a huge problem for multilingual websites.

I used _Polylang Connect for Elementor_ for Polylang working with previous Theme Builder.

**Comment by maurocarreraok** [comment]
Hi.
I dont know why Elementor is not changing the Default Color for Buttons on Woocommerce. If you change Forms and Inputs works... But for everything else is not working.
![image](https://user-images.githubusercontent.com/68023943/90043290-4776c480-dca2-11ea-8910-b02f476f96f6.png)
![image](https://user-images.githubusercontent.com/68023943/90043411-74c37280-dca2-11ea-9ac2-501ef4186070.png)
![image](https://user-images.githubusercontent.com/68023943/90043490-915faa80-dca2-11ea-95cb-b60ba29156b2.png)
![image](https://user-images.githubusercontent.com/68023943/90043526-9f153000-dca2-11ea-86ff-2ed879ef03d3.png)

**Comment by dudaster** [comment]
> > I'm trying to get to the new Theme Builder by pressing CMD+SHIFT+E,
> > I see the E logo, loading and than i get this error
> > https://www.loom.com/share/bfc78bf20b4c4f4f91cc30275d1be3ed
> > ![image](https://user-images.githubusercontent.com/7497319/89624310-b61ce200-d89e-11ea-88ee-014b5eca63c6.png)
> 
> After a quick check, the plugin that cause this bug is:
> **Elementor Custom Skin Pro**
@pintolog It is fixed a new update will arrive soon.

**Comment by jonathansoltero** [comment]
I was thinking, shouldn't there be an option in the theme builder to set a template for mobile, desktop, & tablet separately instead of going into responsive & turning on visibility; not really an issue but could make things easier and much more organized.

**Comment by TomRaraAvis** [comment]
I expected to see #4440 Posts / Archive loop item template designer in theme builder.

**Comment by cebesten** [comment]
Trying to enter the theme builder I get white screen and in the console I see some error in the console, I deactivate all the plugins and let only elementor and E-pro active, same result.
![image](https://user-images.githubusercontent.com/48234458/90104406-15448180-dcf9-11ea-834c-c9c962ff2d43.png)

**Comment by fabianenzensperger** [comment]
> I expected to see #4440 Posts / Archive loop item template designer in theme builder.

Unfortunately, it takes even longer for whatever reason …

**Comment by shilo-ey** [comment]
[**@Rodolphe Bertozzo**](https://github.com/rodolphebertozzo) commented on [Aug 10, 2020, 9:06 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-671504640 "2020-08-10T18:06:07Z - Replied by Github Reply Comments"):
> @shilo-ey To answer you, here are 2 screenshots (header templates), one with the old theme builder and the next one with the new one. We can see that there are 5 headers recognized by Elementor with the old theme constructor and that since the new interface, the 4 other headers are considered as "simple pages".

Yes, I understand the issue, but can you recall how this happened? on upgrade? on creation?



[**@Rodolphe Bertozzo**](https://github.com/rodolphebertozzo) commented on [Aug 10, 2020, 9:09 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-671506468 "2020-08-10T18:09:44Z - Replied by Github Reply Comments"):
> 2 other remarks :
> 
> -Add a line preview option for each model

You can see live preview in the Specific Site Part screen

> -Add a window when you create a template to be able to directly give a name to the template
Thanks



[**@jonathansoltero**](https://github.com/jonathansoltero) commented on [Aug 10, 2020, 11:07 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-671562563 "2020-08-10T20:07:57Z - Replied by Github Reply Comments"):
> How often do the global parts screenshots update? I am still seeing screenshots from yesterday.
> 
> I would remove the Templates tab & put Theme Builder under Elementor tab so it remains clear that it's part of Elementor.

Currently, the screenshots generation triggers once you enter the Theme Builder. So if you create a template, go to Theme Builder - you will see a placeholder, on the second time you will enter the Theme Builder you should see the screenshot if the process succeeded. in the specific site part screen you should see a live preview of it.

This is the trigger to avoid redundant use of resources in your site while Theme Builder is not used.


[**@fabianenzensperger**](https://github.com/fabianenzensperger) commented on [Aug 11, 2020, 12:33 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-671601226 "2020-08-10T21:33:37Z - Replied by Github Reply Comments"):
> @shilo-ey How can I actually change a template created with the Theme Builder? For example converting a "Single Page" template into a "Header" template?
> 
> And what would be MUCH better if header and navigation were separate. So you could leave the navigation to the navigation and place a separate header (for example image with text) on the pages.
> 
> That would make it a lot easier, wouldn't it? Maybe you could think about it for the new Theme Builder. I am happy about your thoughts on it ...

[**@poziomapl**](https://github.com/poziomapl) commented on [Aug 12, 2020, 3:49 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-672850384 "2020-08-12T12:49:29Z - Replied by Github Reply Comments"):
> In version 3.0 Pro in Theme Builder it is not possible to add a translation for a theme page, e.g. with PolyLang or WPML for headers/footers/archives/single pages. Previously, the default Wordpress panel allowed this, and the new Theme Builder layout does not - it is a huge problem for multilingual websites.
> 
> I used _Polylang Connect for Elementor_ for Polylang working with previous Theme Builder.


@fabianenzensperger Didn't quite understand your suggestion, could you please explain?

As for your question - the "Old" Theme Builder will remain present there.


[**@gravnetic**](https://github.com/gravnetic) commented on [Aug 11, 2020, 2:13 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-671635149 "2020-08-10T23:13:38Z - Replied by Github Reply Comments"):
> I have custom classes on objects and I am unable to access the editor now.
> Undefined variable: css
> I am able to access the editor in safe mode.

Sounds like 3rd party plugin conflict. try eliminating the one who causes it by deactivating all plugins and activating them one by one until you can't access the editor again.
Share your findings here please.



[**@maurocarreraok**](https://github.com/maurocarreraok) commented on [Aug 12, 2020, 7:49 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-672989853 "2020-08-12T16:49:23Z - Replied by Github Reply Comments"):
> Hi.
> I dont know why Elementor is not changing the Default Color for Buttons on Woocommerce. If you change Forms and Inputs works... But for everything else is not working.

These settings are deriving from WooCommerce. their selector is "stronger" than Elementor's.

It is on our roadmap to handle these areas.


[**@TomRaraAvis**](https://github.com/TomRaraAvis) commented on [Aug 13, 2020, 9:00 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-673279220 "2020-08-13T06:00:12Z - Replied by Github Reply Comments"):
> I expected to see #4440 Posts / Archive loop item template designer in theme builder.

We will add this feature in the next versions to come, but not until it's perfect. we truly understand the power of Loop Builder and how important it is. please be patient.



[**@cebesten**](https://github.com/cebesten) commented on [Aug 13, 2020, 10:10 AM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-673304340 "2020-08-13T07:10:39Z - Replied by Github Reply Comments"):
> Trying to enter the theme builder I get white screen and in the console I see some error in the console, I deactivate all the plugins and let only elementor and E-pro active, same result.

We will try to reproduce this.

**Comment by rodolphebertozzo** [comment]
@shilo-ey, a simple upgrade

**Comment by oakas33** [comment]
![Screen Shot 2020-08-16 at 11 04 26 AM](https://user-images.githubusercontent.com/69766310/90341164-2370f700-dfb2-11ea-8c2f-18a9cf68f925.png)
![Screen Shot 2020-08-16 at 11 13 51 AM](https://user-images.githubusercontent.com/69766310/90341165-24a22400-dfb2-11ea-8ab7-05ebc30f02b5.png)

I'm having issues with color changes not being reflected on the submit button for forms. Changes are reflected on my website but not reflected in my Elementor Page Editor. It seems to have popped up when multi-step forms came out, though it could be a 3.0 beta bug, I'm not sure. As you can see I've started from scratch and made another form, but I'm having the same issue. Both submit buttons should be set to a black background. (Adjusting the "Previous Button" color settings have no affect on the button color either)


Also, some weird glitch not allowing me to select fields under mailchimp settings (See screenshot) ![Screen Shot 2020-08-16 at 11 05 07 AM](https://user-images.githubusercontent.com/69766310/90341160-148a4480-dfb2-11ea-9335-4a3850d58097.png)

**Comment by black-eye** [comment]
> My Templates > Theme Builder link in Admin Menu reffers to non extisting page (/wp-admin/elementor-app)
> 
> ![obrazek](https://user-images.githubusercontent.com/5634870/89629763-aa312000-d89e-11ea-885d-7e4ab8816831.png)
> 

After some investigation, I've found out it's a conflict with the plugin Admin Menu Editor. I've reported it to the developer.

**Comment by black-eye** [comment]
shilo-ey commented 7 days ago
> [**@black-eye**](https://github.com/black-eye) commented on [Aug 8, 2020, 5:57 PM GMT+3](https://github.com/elementor/elementor/issues/12054#issuecomment-670938473):
> 
> > My first impresions. Lets put aside that after all the hype raised and the time spent waiting I can’t help myself: it’s a bit underwhelming. It’s basicaly just a new look for the old stuff. Yes, I know UI is more than a graphic style, so let me talk the functionality. Because I feel, there are areas, where it got worse.
> > 
> > 1. No direct access to the Theme Buider - admin menu links to 404. Hopefully it’s just a bug.
> 
> Indeed, a bug.
> 
> > 1. No possibility to export all the templates at once or selected in bulk. If there is such possibility, show me it pls.
> 
> You will have to be patient for a couple of more months :)
> 
> > 1. No full template specifications / actions on Theme Builder “dashboard” (all templates filter). I can’t see the instances, conditions etc. When I want to edit the template (what else I’d be there for, right?), I need one extra click to get to the template “detail” and then click the edit link. I usually have only one template per each part, so I’d like to manage them right from the dashboard, not to chase them in several sections one by one.
> 
> This screen is designed for a general overview. the specific part screen should show you a live preview and it's actions.
> 
> > 1. Templates categories are gone.
> 
> If you wish to use categories you can still use the backend screen. although, as you mentioned, when having one part from each type - why would you need it?
> 
> > 1. Why the extra options like export live under “...” button and are not visible right away - there’s plenty of space there.
> 
> It's not an action that you will make on a daily basis and we rather add one more click to make sure you wish to do it.
> 
> > 1. No info about the author, date or shortcode.
> 
> You should see the author name, and the date. can't you see them in the inner screen?
> 
> > 1. Is there a possibility to get to the template dashboard? To set the stuff like language, category etc.
> 
> Please elaborate. what is the template dashboard? WP dashboard post?
> If so, as mentioned, the "Legacy" Theme builder will still be there.
> 
> > 1. I haven’t tried it on multilang site, but I have the feeling it would be a complete disaster. I have no idea, how to select the language there.
> 
> Same as 6.
> 
> > So these are my observations. To be honest, if the current “admin dashboard style” stays in the old place, I will probably use it rather than this new one. Just for the lack of the functionality of the later.
> > Basically the most (and for me only) exiting addition is the template thumbnails. Unfortunately they are not applied to the non-theme templates (yet??), where they would be much more needed.
> 
> Yet.

Most of my concerns were caused by the fact that I wasn't able to access the new Theme Builder correctly, due to the plugin conflict mentioned in the post above.

Now I know how it should work and that the "old" Theme Builder (which I also called "Dashboard") is still accessible. On the other hand, it is accessible rather secretly and not directly via the admin menu (this item now links to the "New" Theme Builder). The only way how to get to the old Theme Builder is to go to My Templates and select All in the filter (correct me, if I'm mistaken). 

And now the things get rather inconstistent:

![obrazek](https://user-images.githubusercontent.com/5634870/90349500-6080cc00-e03a-11ea-8f8f-80dffa360550.png)
If the old Theme Builder should stay in use (and we can clearly see it should because the new Builder is still limited), this navigation and filtering should be made more evident and consistent.

**Comment by hogash** [comment]
I'm struggling to update to the latest beta release but keep getting "An error occurred while updating Elementor Pro: Download failed. Service Unavailable" . Isn't there any direct link?

**Comment by black-eye** [comment]
The Templates thumbnails differ on the dashboard (All Parts) and previews.

In the dashboard there's a desktop layout, but somehow outdated (the actual heading shoud be "Články II"):

![obrazek](https://user-images.githubusercontent.com/5634870/90409739-d53f1f00-e0a9-11ea-9347-ba4d7044534c.png)

The preview shows the mobile view with the updated heading:

![obrazek](https://user-images.githubusercontent.com/5634870/90410165-4ed70d00-e0aa-11ea-8f72-f7f93b4d3303.png)

Moreover, in the dashboard it's shown only the template itself, while the preview is the whole page including the header and footer. This is somehow confusing.

Also please consider adding the previews to the "old" Theme Bulder too. Since they are already created, it shouldn't be the problem. For the non Theme Builder templates you could use the placeholders to distinguish them better.

**Comment by shilo-ey** [comment]
Hey all,

Please check this version thoroughly and report it here.

**This beta3 version includes the following changes:**

- Tweak: Added quick links to Publish toaster message when saving a Site Part
- Tweak: Added Date Modified option to Posts widget metadata
- Tweak: Added a scroll to the part when accessing Specific Site Part screen in Theme Builder
- Tweak: Added ellipsis when there are several conditions in Theme Builder cards
- Tweak: Show "Not found" message on Specific Site Part screen
- Fix: PHP 7.4 compatibility to Media Carousel widget (#11355)
- Fix: Library blocks are missing for some template types on "Create New" action
- Fix: Specific Site Part screen shows previews as responsive view instead of desktop in Theme Builder
- Fix: Specific Site Part live preview interactions are not disabled in Theme Builder
- Fix: Titles are not capitalized in Theme Builder
- Fix: Various issues with screenshots mechanism
- Fix: Show when arriving from search engines rule doesn’t work in Popup
- Fix: Child categories are shown with a different parent category in Query control


**Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/pojome/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**

Thanks again to everyone.

**Comment by fabianenzensperger** [comment]
Thank you for the update. I noticed 2 things in the beta:

1. If you save the conditions with "Save & Close", the module will not be terminated. I think this could be confusing because you think nothing is happening? It would be better if you came back to the template or the previous view ...

<img width="1680" alt="Bildschirmfoto 2020-08-18 um 22 06 50" src="https://user-images.githubusercontent.com/57997262/90566342-98a61d00-e1a8-11ea-9163-72546392ac68.png">


2. Since Wordpress 5.5 and since Pro Beta 2 and now also in Beta 3, a black, transparent screen appears before the Theme Builder loads. As far as I can remember, this was not the case with Pro Beta 1 and before Wordpress 5.5. That's nothing bad, but it's just not so nice ...

<img width="1680" alt="Bildschirmfoto 2020-08-18 um 22 08 56" src="https://user-images.githubusercontent.com/57997262/90566360-9fcd2b00-e1a8-11ea-8859-bb12f5509c95.png">


Otherwise great update. Does the search still come in the new theme builder? Goodnight!

**Comment by black-eye** [comment]
This one may sound ridiculous, but when I'm creating a new template using the new Template Builder, why there's no dialog box asking me for the name and the details like it was before?

When you want to name the template you've just created, you have to return to the main panel and actualy rename it from an autogenerated title. Really inconvenient.  Is it a bug or an intention I don't understand?

This is the part of the workflow, I'm missing:

![obrazek](https://user-images.githubusercontent.com/5634870/90617873-80b5b400-e20f-11ea-94d6-4a32dd478208.png)

**Comment by Worldwebscapes** [comment]
> This one may sound ridiculous, but when I'm creating a new template using the new Template Builder, why there's no dialog box asking me for the name and the details like it was before?
> 
> When you want to name the template you've just created, you have to return to the main panel and actualy rename it from an autogenerated title. Really inconvenient. Is it a bug or an intention I don't understand?
> 
> This is the part of the workflow, I'm missing:
> 
> ![obrazek](https://user-images.githubusercontent.com/5634870/90617873-80b5b400-e20f-11ea-94d6-4a32dd478208.png)

I agree with Black-Eye here. I like to name my templates when starting them. Having to go to the gear settings panel to name my new template is more difficult. It is also not easy to edit the permalink slugs in traditional ways.

**Comment by rodolphebertozzo** [comment]
Has a solution been found for the shortcodes?

**Comment by cake1123** [comment]
#1284 Please add this, It's a long time coming and a very much needed feature.

**Comment by fabianenzensperger** [comment]
> #1284 Please add this, It's a long time coming and a very much needed feature.

Unfortunately, Elementor usually takes a long time for such simple things …

**Comment by TomRaraAvis** [comment]
**Elementor Beta 6 + Elementor Pro Beta 3**

I'm getting these weird images in my media library. I don't know whether to post in Beta 6 or Pro Beta 6, so I'm doing both. Anyone else getting these?

![weird-media-library](https://user-images.githubusercontent.com/1435355/91123520-136cad80-e66b-11ea-9a34-5e445eda4a09.png)

**Comment by fabianenzensperger** [comment]
> **Elementor Beta 6 + Elementor Pro Beta 3**
> 
> I'm getting these weird images in my media library. Anyone else getting these?
> 
> ![weird-media-library](https://user-images.githubusercontent.com/1435355/91123520-136cad80-e66b-11ea-9a34-5e445eda4a09.png)

Yes, same here.

**Comment by shilo-ey** [comment]
Hey guys, thanks again for sharing your inputs and thoughts!

The inputs shared were extremely helpful.

We started rolling out this version of Elementor Pro to all of our users and it will be updated soon in the your dashboards.

If some of you hasn't got it yet, try again in couple of hours.

Before upgrading please make sure you backup your site, due to the fact that this is a major verison.

As for issues not being taken care of yet - it will be fixed in the upcoming days, so if you find anything else - do share it with us in a proper issue report.

Elementor Pro 3.0 requires Elementor 3.0 core version to run, make sure your site's cache is cleared and 3rd party plugins and themes are compatible with Elementor 3.0

@fabianenzensperger @black-eye 
Appreciate your suggestions and we will evaluate it for later versions

@TomRaraAvis - these are the auto screenshots created for the Theme Builder.

Thanks again everyone!

**Comment by fabianenzensperger** [comment]
> > This one may sound ridiculous, but when I'm creating a new template using the new Template Builder, why there's no dialog box asking me for the name and the details like it was before?
> > When you want to name the template you've just created, you have to return to the main panel and actualy rename it from an autogenerated title. Really inconvenient. Is it a bug or an intention I don't understand?
> > This is the part of the workflow, I'm missing:
> > ![obrazek](https://user-images.githubusercontent.com/5634870/90617873-80b5b400-e20f-11ea-94d6-4a32dd478208.png)
> 
> I agree with Black-Eye here. I like to name my templates when starting them. Having to go to the gear settings panel to name my new template is more difficult. It is also not easy to edit the permalink slugs in traditional ways.

@shilo-ey Thank you for your final feedback. But why was the workflow changed or the module removed in Beta 3? It wasn't like that in Beta 1 & 2, and it was actually perfect! Anyway, thank you, I hope it will come back.

**Comment by black-eye** [comment]
> **Elementor Beta 6 + Elementor Pro Beta 3**
> 
> I'm getting these weird images in my media library. Anyone else getting these?
> 
> ![weird-media-library](https://user-images.githubusercontent.com/1435355/91123520-136cad80-e66b-11ea-9a34-5e445eda4a09.png)

@TomRaraAvis, good point. I don't like this behaviour at all, I like to keep my media library clean and organized, not to be bloated with some "system" files. 

@shilo-ey, please consider moving the files to a private folder, similarly to page CSS.

@TomRaraAvis Since this thread is closed now, would you pls consider opening a new issue for it?
